hummingbird.py

```diff
@@ -0,0 +1,153 @@
+from os import remove
```

Comment:
I like to order my imports like this:
```
python library imports

third party library imports

imports from this project
```

so that it looks something like this:
```python
import datetime
from sys import stdin

from django.conf import settings
import requests

import config
from utils import log
```

Can we change the format to be this way? (I also order the sections alphabetically).